sources/packages/backend/apps/db-migrations/tsconfig.app.json

```diff
@@ -2,7 +2,6 @@
   "extends": "../../tsconfig.json",
   "compilerOptions": {
     "declaration": false,
-    "sourceMap": false,
```

Comment:
Was it removed because by default it is already false?

Reply:
no if we have this sourceMap defined here, it will override the tsconfig.json configuration. The default configuration is true in our tsconfig, so to ignore the overriding and generate the map.js files, I have removed it here.

Reply:
Thanks for reverting the file.